Doc/reference/expressions.rst

```diff
@@ -478,8 +478,8 @@ allowing any pending :keyword:`finally` clauses to execute.
 .. index::
    single: from; yield from expression
 
-When ``yield from <expr>`` is used, it treats the supplied expression as
-a subiterator. All values produced by that subiterator are passed directly
+When ``yield from <expr>`` is used, the supplied expression must be an
+iterable. The values produced by iterating that iterable are passed directly
 to the caller of the current generator's methods. Any values passed in with
```

Comment:
Is "the current generator's methods" the correct term to use here?  (Should it be "the current generator method" or something like that?)

Reply:
Good question.  I believe the answer is yes, this part of the doc was and is correct.  The 'current generator' is the one with 'yield from'.  The two methods for getting yielded values are `__next__` and `send`.

Reply:
`methods` could be expanded to `'__next__' or 'send' method.` (with markup).  I notice that the two methods to pass things into the generator are explicitly named in the next sentence.  What do you think?

Reply:
I see what you mean.  Though I'm not sure "passed directly to the caller of ... " is that clear either. Maybe this whole sentence can be rephrased/simplified?

Reply:
"The values produced by iterating that iterable form part of the current generator's output."  or some such?

Reply:
I want to stick with the defined scope of the issue, to replace the ambiguous or incorrect 'subiterator' in these two sentences, so it is clear what one can pass to 'yield from'. Do you have any comment on my replacements?.  After sleeping on them, I think them good enough and am ready to merge.  

I have also wondered about the *exact* meaning of 'passed directly', but the visible effect would what lines of code appear in a trace.  Does 'passed directly' mean that the 'yield from' line would *not* appear?  I suspect so; but if I cared I would run experiments.  I consider this a separate issue.

Reply:
Yes, the part you changed looks fine.